config/settings.yml

```diff
@@ -1200,6 +1203,7 @@
       :storage_file_collection_interval: 1.days
       :storage_file_collection_time_utc: 21600
       :task_timeout_check_frequency: 1.hour
+      :vim_performance_states_purge_interval: 1.day
```

Comment:
Does this seem like a good value?

Reply:
Thought we'd do something closer to 6 month. but I not know the usage of this table.